src/test/java/emissary/test/core/junit5/RegressionTest.java

```diff
@@ -51,7 +51,7 @@ public abstract class RegressionTest extends ExtractionTest {
     /**
      * The list of actual logEvents generated by executing the place.
      */
-    private List<SimplifiedLogEvent> actualSimplifiedLogEvents;
+    protected List<SimplifiedLogEvent> actualSimplifiedLogEvents;
```

Comment:
Please correct me if I am wrong but I don't see a need for the modifiers in this class to be modified to support the SBC length attribute in the XML